index.html

```diff
@@ -3,111 +3,106 @@
 title: Your New Jekyll Site
 ---
 
-<div id="info" class="clearfix">
+<section class="info">
 
-    <div id="screens">
-        <a href="/screenshots/" title="view more screenshots of i3">
-        <img id="screenshot" src="/screenshots/i3-9.bigthumb.png" width="375" height="234" alt="i3 screenshot"
-	 srcset="/screenshots/i3-12.bigthumb.jpg 2x">
+    <div class="info-text">
+        <h2>Do What I Mean. Good Docs. Clean Code. Sounds good?</h2>
+        <p>
```

Comment:
There’s almost no space between heading, paragraph and download button. I think the stylesheet needs explicit `margin` definitions for basic elements such as headings and paragraphs.

Reply:
a7983848b6c3c1a7d7eb151b9cd3da259e2ffeac